### PR TITLE
[MediaBundle] Fix for removing media files and directories 

### PR DESCRIPTION
**Pull request by Minhan12**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets |

This pull request will fix 2 bugs in the MediaBundle.

1. When using the command `php bin/console kuma:media:clean-deleted-media --force` to delete the files by the original filename, it will skip the files from the filesystem that have been renamed due to strange characters in the filename. Using the command `php bin/console kuma:media:rename-soft-deleted -o` will update the filename back to the original filename. However, it doesn't use the same method as when an user uploads a file so this command will not update the filename correctly.
This fix will add the slugifier to the command so it will pass the same method when renaming the filename.

2. The command also expects the directory to be deleted. however, it uses a directory name to check if a file exists, so it fail on the check so it doesn't delete the directory.

### CODE REVIEW

**Review by ProfessorKuma — CHANGES_REQUESTED** [review]
Hi @, your PR needs some changes
* It seems that our checklist is missing or incomplete

**Review by ProfessorKuma — CHANGES_REQUESTED** [review]
Hi @, your PR needs some changes
* It seems that you should have submitted to the latest minor branch.
* This PR seems to need a milestone of a patch release.

**Review by acrobat — CHANGES_REQUESTED** [review]
Thanks for the pr @Minhan12! Can you also rebase this PR against the 5.6 branch? Thanks!

I've also left a few comments

File: src/Kunstmaan/MediaBundle/Helper/File/FileHandler.php [hunk 1]
```diff
@@ -230,7 +230,7 @@ public function removeMedia(Media $media)
 
         // Remove the files containing folder if there's nothing left
         $folderPath = $this->getFileFolderPath($media);
-        if ($adapter->exists($folderPath) && $adapter->isDirectory($folderPath) && !empty($folderPath)) {
+        if ($adapter->isDirectory($folderPath) && !empty($folderPath)) {
```
[the remark this answers is not in the document]
Comment (Minhan12) [reply src/Kunstmaan/MediaBundle/Helper/File/FileHandler.php:233]: The isDirectory will check if the directory exists with the method `ensureDirectoryExists` from the FlySystem.

So the call to the method exists() seems double

**Pull request closed without merging**